cmd/virtctl/virctl.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash 
+
+# This script is designed to be proxy used to open virtctl from config file
+
+set -ex
+
+function usage() {
+  echo "Illegal usage."
+  echo "Use ${0} config_file."
+  echo "Config file containes single line: {vnc|console} vm_name<newline>"
```

Comment:
I still think the literal `<newline>` is confusing. that usually means to the user "replace newline with something appropriate"... :) and in fact, this isn't needed. Try adding trailing whitespace to your VM name. It gets stripped.

Reply:
ok agree, double checked and it works without the new line :). Thanks for help